packages/manager/modules/bm-server-components/src/network-tile/controller.js

```diff
@@ -10,6 +10,7 @@ export default class BmServerComponentsNetworkTileController {
   }
 
   $onInit() {
+    this.statePrefix = this.statePrefix || 'app.dedicated-server.server';
```

Comment:
this is not necessary, it will work without it

Reply:
It's applied for all places where i use child state.